examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/EventHubService.cs

```diff
@@ -74,9 +76,14 @@ private Task ProcessEventHandler(ProcessEventArgs eventArgs)
         {
             _logger.LogInformation("EventHub update received. Triggering cache invalidation.");
 
-            //
-            // Set the cached value for key-values registered for refresh as dirty.
-            _configurationRefresher?.SetDirty();
+            // Build EventGridEvent from notification message
+            EventGridEvent eventGridEvent = EventGridEvent.Parse(BinaryData.FromBytes(eventArgs.Data.EventBody));
```

Comment:
`eventArgs.Data.EventBody` is already `BinaryData` type. `BinaryData.FromBytes` is not required.

Reply:
Updated. Thanks!